source/_addons/samba.markdown

```diff
@@ -55,4 +62,9 @@ allow_hosts:
   required: false
   default: '`["10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]`'
   type: list
+veto_files:
+  description: List of files that are neither visible nor accessible. Useful to stop clients from littering the share with temporary hidden files (ex: macOS .DS_Store, Windows Thumbs.db)
```

Comment:
```suggestion
  description: List of files that are neither visible nor accessible. Useful to stop clients from littering the share with temporary hidden files (i.e., macOS .DS_Store, Windows Thumbs.db)
```

Reply:
ex: or e.g. is grammatically correct here. i.e. should not be used.

Reply:
Then change it to `e.g.` 😉

Reply:
Changed to e.g. per request